pkg/config/dict.go

```diff
@@ -30,6 +30,15 @@ type Dict struct {
 	m map[string]cty.Value
 }
 
+// NewDict constructor
+func NewDict(m map[string]cty.Value) Dict {
```

Comment:
It looks to me like the code (e.g. `Set`) is safe against a blueprint with empty `vars` block. But I'd like to raise possibility here of using `NewDict(empty_map)` at some point if there is any reason.

If you believe there is no reason to do so, please close this comment.

Reply:
We are good.
```go
NewDict(nil) == NewDict(map[string]cty.Value{}) == Dict{}
```